From c028c2bcdbe7b53df8be44919bbad82061329913 Mon Sep 17 00:00:00 2001
From: h2zero <powellperalta@gmail.com>
Date: Fri, 24 Apr 2020 08:25:42 -0600
Subject: [PATCH] nimble/gap: Make initial connection event length constants
 zero.

The NimBLE controller currently does not use these parameters, however other controllers such as the esp32 do make use of them.

This PR is for changing these to a more "neutral" value to avoid unexpected behavior when using a different controller.
---
 nimble/host/include/host/ble_gap.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/host/ble_gap.h b/src/host/ble_gap.h
index 271340f..b4dbdb0 100644
--- a/src/host/ble_gap.h
+++ b/src/host/ble_gap.h
@@ -98,8 +98,8 @@ struct hci_conn_update;

 #define BLE_GAP_INITIAL_CONN_LATENCY        0
 #define BLE_GAP_INITIAL_SUPERVISION_TIMEOUT 0x0100
-#define BLE_GAP_INITIAL_CONN_MIN_CE_LEN     0x0010
-#define BLE_GAP_INITIAL_CONN_MAX_CE_LEN     0x0300
+#define BLE_GAP_INITIAL_CONN_MIN_CE_LEN     0x0000
+#define BLE_GAP_INITIAL_CONN_MAX_CE_LEN     0x0000

 #define BLE_GAP_ROLE_MASTER                 0
 #define BLE_GAP_ROLE_SLAVE                  1
